7-gce/config.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Remove this file, and add a step to the guide similar to this one: https://cloud.google.com/nodejs/getting-started/using-cloud-sql#configuring_settings